plugins/catalog-import/src/components/ImportComponentPage.tsx

```diff
@@ -64,8 +80,10 @@ function repositories(configApi: ConfigApi): string[] {
 
 export const ImportComponentPage = ({
   catalogRouteRef,
+  integrationConfig,
 }: {
   catalogRouteRef: RouteRef;
+  integrationConfig: IntegrationConfig;
```

Comment:
Tbh I think we should just put `enablePullRequest` straight into the props rather than the extra layer. We should also make the prop optional and default to true, removing the need to specify it at all if you want the default behaviour.

Reply:
Thanks, I assume this needs to be propagated to the [Router level](https://github.com/backstage/backstage/pull/4331/files#diff-875d76559612aeab3a162682e160e8aa49e6d395272ac1f0c7146a9284d83b40R31) as well.

Happy to remove the extra layer, you know the codebase far better than me 😄